src/app/tests/suites/certification/Test_TC_DGGEN_2_1.yaml

```diff
@@ -220,7 +229,9 @@ tests:
     - label: "Step 4c:Reboot target device(DUT)"
       PICS: PICS_USER_PROMPT
       verification: |
-          Did the DUT successfully reboot?
+          execute the below mentioned command to put DUT into a commissionable state, Pls use equivalent command on the respective DUT
+
+          After reboot launch the app  ./chip-all-clusters-app --KVS ./mytest.txt
```

Comment:
The correct "reboot" on an RPi is to just kill the app and restart it with the same KVS.  It does not involve rebooting the RPi.

Reply:
Reverted back the changes.